standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/model/MScheduledExecution.java

```diff
@@ -0,0 +1,170 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hive.metastore.model;
+
+import org.apache.hadoop.hive.metastore.api.QueryState;
+import org.apache.hadoop.hive.metastore.api.ScheduledQuery;
+import org.apache.hadoop.hive.metastore.api.ScheduledQueryKey;
+
+public class MScheduledExecution {
+
+  public MScheduledExecution() {
+  }
+
+  private int scheduledExecutionId;
+  private MScheduledQuery scheduledQuery;
+  private String executorQueryId;
+  private String state;
+  private Integer startTime;
+  private Integer endTime;
+  private String errorMessage;
+  private Integer lastUpdateTime;
+
+  @Override
+  public String toString() {
+    return String.format("state: %s, scheduledQuery: %s, execId: %d", state, scheduledQuery.getScheduleName(),
+        executorQueryId);
+  }
+
+  //  public MScheduledQuery2(ScheduledQuery s) {
```

Comment:
We can remove commented code?

Reply:
![x](https://media.giphy.com/media/7SOdwNYHzCI7ygrLw7/giphy.gif)